### ISSUE TITLE
Node memory capacity incorrect on graph

### ISSUE THREAD

**Issue by geruetzel** [lensapp/lens#370]
**Describe the bug**
The value for the overall capacity of memory of all kubernetes nodes is not correct on some cases. This only applies to the graph itself, in the table below the graph, the 'Capacity' is correct.

**To Reproduce**
I don't know how you can reproduce the problem, I only get it on one of my 4 clusters.

**Expected behavior**
For lens to show the correct memory capacity graph for a kubernetes node.

**Screenshots**
<img width="1570" alt="Screenshot 2020-05-13 at 14 21 19" src="https://user-images.githubusercontent.com/7011479/81812046-dbab8100-9525-11ea-986a-04cd2aed6de7.png">
<img width="645" alt="Screenshot 2020-05-13 at 14 32 20" src="https://user-images.githubusercontent.com/7011479/81812592-a5223600-9526-11ea-86b7-0ec8d8b2b380.png">

As you can see, the graph and the tooltip show about 94 GB of available memory for that node, which is wrong, as you can see in the table below the graph.

**Environment (please complete the following information):**
- Lens Version:
- OS: OSX
- Installation method: Image from Github

**Comment by TweetyHH** [comment]
Same here and on the cluster View too.

Its on CPU Requests, CPU Limits and CPU Capacity, Memory Requested, Memory Limits, Memory Capacity.

It,s always factor 3, but not for CPU/Memory usage.

**Environment:** 
OS: Windows
Version: 3.3.1/3.4.0
Cluster: Kubernetes on Azure (AKS), Version 1.5.7
Prometheus/Monitoring: fresh installed from lens (3.3.1)

**Comment by geruetzel** [comment]
@TweetyHH yes, you are right. the capacity on cluster view is incorrect for me as well. my total available cluster resources should be

72 cpu
141 GB ram
330 Pods

but is:
<img width="671" alt="Screenshot 2020-05-13 at 15 08 05" src="https://user-images.githubusercontent.com/7011479/81816415-9722e400-952b-11ea-8ed7-e046f0f55633.png">

so only cpu is correct for me. ram and pods should be only a third of the given value. (i guess divided by the amount of available nodes?)

**Comment by TweetyHH** [comment]
@geruetzel I don't thinks its about number of the cluster nodes (worker). We have here 7 or 8 nodes. The same issue is on the diagrams of a pod. 

Additional Question: Would it be possible to show the current usage of cpu/memory the pod view. On the node view is an indication of the current cpu/memory usage.

**Comment by matthewhembree** [comment]
I am seeing something similar on deployment (pod) resource requests/limits; if set. It's off by a factor of 3. I see this on one of my clusters, but not the others. Lens will display a different value than what `kubectl` will report.

**Comment by jakolehm** [comment]
@geruetzel @TweetyHH @matthewhembree could you check if prometheus has collected duplicate data (via Prometheus web UI or similar)?

**Comment by TweetyHH** [comment]
@jakolehm It looks like there is some kind of tripple data in the lens prometheus instance. In our setup we have our own prometheus/monitoring stack running, but additionally installed the lens prometheus/monitoring stack. I think they should work independently.

Here a screenshot for one node of kube_node_status_allocatable_cpu_cores

![image](https://user-images.githubusercontent.com/626865/83451175-d8395480-a456-11ea-9e4d-5602a35bcf3d.png)

[edited better screenshot]

**Comment by hardenchant** [comment]
I dont know related my problem to this or not, but i have wrong memory capacity at nodes list, annoys me a lot.
![image](https://user-images.githubusercontent.com/17965301/126145239-0ab273a9-fabd-471f-9d0c-d0dd4bbdb1ce.png)
![image](https://user-images.githubusercontent.com/17965301/126145345-f56fd6ac-102f-437f-b904-eb555b1e4265.png)

But on memory usage prometheus view everything is fine.
![image](https://user-images.githubusercontent.com/17965301/126145532-f362806f-aece-4658-8f3b-22f7881ee73b.png)

Lens 5.1.1-latest.20210716.1
MacOS Big Sur

**Comment by zerodayyy** [comment]
Having the same issue, however besides node metrics, pod/container metrics are affected as well. There are no duplicates in Prometheus, however this is what happens in Lens:

<img width="815" alt="image" src="https://user-images.githubusercontent.com/49662783/147682439-fb9e00ef-52ff-4748-81a0-14e0d9d33683.png">

As you can see, at first requests and limits are displayed correctly, however the next data point comes multiplies by 2.
